src/containers/ManageListingsPage/ManageListingsPage.duck.js

```diff
@@ -143,6 +166,30 @@ const manageListingsPageReducer = (state = initialState, action = {}) => {
       };
     }
 
+    case DISCARD_DRAFT_REQUEST:
+      return {
+        ...state,
+        discardingDraft: payload.listingId,
```

Comment:
ManageListingPage.js: has something like this:
`actionsInProgressListingId={openingListing || closingListing}`
That probably should handle **discardingDraft** too.

Reply:
Included!